src/puppetlabs/ezbake/dependency_utils.clj

```diff
@@ -116,6 +117,23 @@
         (assoc acc project-name (.getInputStream jar-file jar-entry)))
       acc)))
 
+(defn get-stream-from-project-jar
+  [lein-project file-path]
+  (let [project-jar-file (fs/file (:target-path lein-project)
+                                  (str
+                                   (:name lein-project)
+                                   "-"
+                                   (:version lein-project)
+                                   ".jar"))]
+    (if (fs/exists? project-jar-file)
+      (let [project-jar (JarFile. project-jar-file)]
+        (if-let [project-jar-entry (find-file-in-jar project-jar
+                                                     file-path)]
+          (.getInputStream project-jar project-jar-entry)))
+      (throw
```

Comment:
For leiningen it is better to use `leiningen.core.main/abort` as shown here: https://github.com/puppetlabs/ezbake/blob/master/src/puppetlabs/ezbake/exec.clj#L35